fedireads/incoming.py

```diff
@@ -128,9 +131,13 @@ def verify_signature(request):
 @app.task
 def handle_follow(activity):
     ''' someone wants to follow a local user '''
-    # figure out who they want to follow
-    to_follow = models.User.objects.get(actor=activity['object'])
-    # figure out who they are
+    # figure out who they want to follow -- not using get_or_create because
+    # we only allow you to follow local users
+    try:
+        to_follow = models.User.objects.get(actor=activity['object'])
+    except models.User.DoesNotExist:
```

Comment:
Why don't we just propagate the exception here? The caller doesn't do anything with the return value, so I think it makes debugging easier to just raise the exception.